core/src/main/kotlin/com/malinskiy/marathon/execution/Scheduler.kt

```diff
@@ -1,12 +1,16 @@
 package com.malinskiy.marathon.execution
 
-import com.malinskiy.marathon.aktor.Aktor
 import com.malinskiy.marathon.analytics.Analytics
 import com.malinskiy.marathon.device.DevicePoolId
 import com.malinskiy.marathon.device.DeviceProvider
-import com.malinskiy.marathon.healthCheck
+import com.malinskiy.marathon.execution.DevicePoolMessage.FromScheduler
+import com.malinskiy.marathon.execution.DevicePoolMessage.FromScheduler.AddDevice
+import com.malinskiy.marathon.execution.DevicePoolMessage.FromScheduler.RemoveDevice
 import com.malinskiy.marathon.test.Test
+import com.malinskiy.marathon.waitWhileTrue
+import kotlinx.coroutines.experimental.channels.SendChannel
 import kotlinx.coroutines.experimental.launch
+import mu.KotlinLogging
 
 /**
```

Comment:
this comment belongs to Scheduler(not a const), and i think it's better to write more about relations between strategies (like in corresponding issues)